### PR TITLE
Added err msg frequency limit

### PR DESCRIPTION
**Pull request by averybigant**

### CODE REVIEW

File: exampleConfig.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ var foo = {
   ignore: function(line) {
     return line.indexOf('to-be-ignored') < 0;
   },
+  limit: 0,
```
Comment (hit9) [inline exampleConfig.js]: what about speedLimit? or sth. else..

File: index.js [hunk 2]
```diff
@@ -29,9 +29,27 @@ function watch(target) {
   target.extract = target.extract || function(line) {
     return line;
   };
+  target.limit = target.limit || 0;
```
Comment (hit9) [inline index.js]: 0 is no limit ?

Comment (averybigant) [inline index.js]: 恩,不过默认可以搞个大点的限制我觉得,几百几千的.

File: exampleConfig.js (hunk 1 above)
Comment (averybigant) [inline exampleConfig.js]: frequencyLimit?

File: index.js [hunk 3]
```diff
@@ -29,9 +29,27 @@ function watch(target) {
...
+  setInterval(function() {
+      lastCount = lineCount;
+      lineCount = 0;
+      if(banned && lastCount < target.limit / 2){
+          banned = false;
+          emitter.emit('alert', 2, [],
+                       util.format('error msg frequency goes below %d/s again',
+                       target.limit));
+      }
+  }, 500);
```
Comment (hit9) [inline index.js]: .5s 执行一次？

File: exampleConfig.js (hunk 1 above)
Comment (hit9) [inline exampleConfig.js]: ok

**Pull request merged**